docs/ros-snap.md

```diff
@@ -349,15 +349,13 @@ you'll have a .snap.
 
 ### Take the .snap for a test drive
 
-You can transfer your newly-minted .snap to your Ubuntu Core machine and install
-it at the same time via `snappy-remote`, for example:
+Transfer your newly-minted .snap to your Ubuntu Core machine with `scp`,
+then install it:
 
```

Comment:
Ah, that sounds very desktop-specific. I guess we need to come up with documentation that works for both now, eh?